install

```diff
@@ -29,11 +29,12 @@ sudo rm -f /etc/default/emonhub
 sudo ln -s $GIT_PATH/conf/default/emonhub /etc/default/emonhub
 
 ### create folder and move settings file (unless it exists already)
```

Comment:
also if you're not deleting but commenting out please put in a comment saying why into the code.